test/Infrastructure.IntegrationTest/AdminConsole/Repositories/OrganizationUserRepository/OrganizationUserRepositoryTests.cs

```diff
@@ -541,6 +541,74 @@ public async Task GetManyDetailsByUserAsync_Works(IUserRepository userRepository
         Assert.Equal(organization.UseAdminSponsoredFamilies, result.UseAdminSponsoredFamilies);
     }
 
+    [Theory, DatabaseData]
+    public async Task GetManyDetailsByUserAsync_ShouldPopulateSsoPropertiesCorrectly(
```

Comment:
I also think a general test would be worthwhile, which checks all properties, similar to the provider test.

Reply:
`GetManyDetailsByUserAsync_Works` already did that, I added the missing properties to it